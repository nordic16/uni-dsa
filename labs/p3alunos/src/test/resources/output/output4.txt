[TIME: 0] Printer 0 online, ready to take jobs.
[TIME: 0] Printer 1 online, ready to take jobs.
[TIME: 0] Job job1 (NORMAL) scheduled to printer 0, printing will take 25.
[TIME: 0] Job job2 (HIGH) scheduled to printer 1, printing will take 5.
[TIME: 0] Job job3 (HIGH) scheduled to printer 0, printing will take 10.
[TIME: 0] Job job4 (HIGH) scheduled to printer 1, printing will take 15.
[TIME: 25] Job jobf (NORMAL) scheduled to printer 1, printing will take 10.
All print jobs have been assigned to a printer!
[TIME: 30] Job job2 has finished printing. Total wait time: 30.
[TIME: 10] Job job3 has finished printing. Total wait time: 10.
[TIME: 45] Job job4 has finished printing. Total wait time: 45.
[TIME: 55] Job jobf has finished printing. Total wait time: 30.
[TIME: 35] Job job1 has finished printing. Total wait time: 35.
All printing jobs have been processed!