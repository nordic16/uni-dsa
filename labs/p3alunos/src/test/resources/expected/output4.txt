[TIME: 0] Printer 0 online, ready to take jobs.
[TIME: 0] Printer 1 online, ready to take jobs.
[TIME: 0] Job job1 (NORMAL) scheduled to printer 0, printing will take 25.
[TIME: 0] Job job2 (HIGH) scheduled to printer 1, printing will take 5.
[TIME: 0] Job job3 (HIGH) scheduled to printer 0, printing will take 10.
[TIME: 0] Job job4 (HIGH) scheduled to printer 1, printing will take 15.
[TIME: 5] Job job2 has finished printing. Total wait time: 5.
[TIME: 10] Job job3 has finished printing. Total wait time: 10.
[TIME: 20] Job job4 has finished printing. Total wait time: 20.
[TIME: 25] Job jobf (NORMAL) scheduled to printer 1, printing will take 10.
All print jobs have been assigned to a printer!
[TIME: 35] Job job1 has finished printing. Total wait time: 35.
[TIME: 35] Job jobf has finished printing. Total wait time: 10.
All printing jobs have been processed!